draft-js-plugins-editor/src/Editor/index.js

```diff
@@ -63,14 +63,17 @@ class PluginEditor extends Component {
   }
 
   componentWillReceiveProps(next) {
-    if (
-      this.props.editorState.getDecorator() !== null &&
-      this.props.editorState.getDecorator() !== next.editorState.getDecorator()
-    ) {
-      const decorator = this.props.editorState.getDecorator();
-      const editorState = EditorState.set(next.editorState, { decorator });
-      this.onChange(moveSelectionToEnd(editorState));
-    }
+    const curr = this.props;
+    const currDec = curr.editorState.getDecorator();
+    const nextDec = next.editorState.getDecorator();
+
+    if (currDec === nextDec) return;
+    if (currDec && nextDec && currDec.decorators.size === nextDec.decorators.size) return;
```

Comment:
The important change is this one (I think), previously we would check `currDec !== nextDec` but that returns true even though they're the same in our app. (don't ask me why _that_ is, but this is an easier fix than figuring that out and will keep others from running into that problem)

By checking the size explicitly we make sure we only reinject them when they've changed.

Reply:
well if it's not exactly the same object it'll fail, so my guess is you somehow recreate the decorator? meh